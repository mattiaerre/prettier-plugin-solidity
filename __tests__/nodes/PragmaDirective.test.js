const PragmaDirective = require('../../src/nodes/PragmaDirective');

test('nodes list to match snapshot', () => {
  expect(
    PragmaDirective.print({
      node: {
        name: 'solidity',
        value: "not a valid range but shouldn't be replaced with an null value."
      }
    })
  ).toMatchSnapshot();
});
