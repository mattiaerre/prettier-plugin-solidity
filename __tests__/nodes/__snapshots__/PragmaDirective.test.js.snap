// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nodes list to match snapshot 1`] = `
Object {
  "parts": Array [
    "pragma ",
    "solidity",
    " ",
    "not a valid range but shouldn't be replaced with an null value.",
    ";",
  ],
  "type": "concat",
}
`;
